src/config.c

```diff
@@ -1222,6 +1201,46 @@ char* bud_config_encode_npn(bud_config_t* config,
 #endif  /* OPENSSL_NPN_NEGOTIATED */
 
 
+bud_error_t bud_config_load_file(bud_config_t* config,
+                                 const char* path,
+                                 const char** out) {
+  bud_error_t err;
+  int fd;
+  char* content;
+
+  /* Check if we already have cache entry */
+  content = bud_hashmap_get(&config->files.hashmap, path, strlen(path));
+  if (content != NULL) {
+    *out = content;
```

Comment:
@indutny a random question here:  In the future shall we have the ability to reload files ie if a cert changes or a configuration changes and we don't want to kill bud?